test/ShapeCrawler.Tests.Unit/ShapeCollectionTests.cs

```diff
@@ -445,6 +445,52 @@ public void AddPicture_too_large_adds_picture()
         pres.Validate();
     }
 
+    [Test]
+    public void AddPicture_adds_svg_picture_no_width_height_tags()
+    {
+        // Arrange
+        var pres = new Presentation();
+        var shapes = pres.Slides[0].Shapes;
+        var image = TestHelper.GetStream("test-vector-image-wide.svg");
+        image.Position = 0;
+
+        // Act
+        shapes.AddPicture(image);
+
+        // Assert
+        shapes.Should().HaveCount(1);
```

Comment:
Please remove this assertion. I believe it has little use in this test.

It is generally recommended to use as less assertion statements as possible in a unit test. Thus, for example, in this test, we focus more on the fact that this test is about the size of the added picture.

Reply:
is done